api/python/arrus/utils/imaging.py

```diff
@@ -3632,7 +3632,7 @@ def prepare(self, const_metadata):
 
         self.n_elements = probe_model.n_elements
         burst_factor = pulse.n_periods / (2 * self.fn)
-        self.initial_delay = -start_sample / 65e6 + burst_factor
+        self.initial_delay = -start_sample / 120e6 + burst_factor
```

Comment:
Sampling frequency should depend on AFE(OEM) variant

Reply:
Fixed